src/components/Tabs.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cn from 'classnames';
+import PropTypes from 'prop-types';
+
+import './Tabs.scss';
+
+const Tabs = ({ tabs, currentTab, choiceTab }) => (
+  <div className="tabs">
+    <div className="tabs__container">
+      {tabs.map(tab => (
+        /* eslint-disable */
+        <a
```

Comment:
and eslint will be okay

```suggestion
        <button
```